src/components/Typography/Typography.types.ts

```diff
@@ -0,0 +1,76 @@
+import { TooltipProps } from 'antd'
+
+/**
+ * @link https://ant.design/components/typography#copyable
+ */
+export type CopyConfig = {
+
+  /**
+   * The text to copy.
+   */
+  text?: string;
+
+  /**
+   * Function called when the text is copied.
+   */
+  onCopy?: (event?: React.MouseEvent<HTMLDivElement>) => void;
+
+  /**
+   * Custom copy icon(s): [copyIcon, copiedIcon].
+   */
+  icon?: React.ReactNode;
+
+  /**
+   * Custom tooltip text(s): [copyText, copiedText]. If the attribute is set to false, no tooltip will be shown.
```

Comment:
It is not clear if both a single icon and an array of icons can be passed, I would try to specify it.